app/assets/stylesheets/application.scss

```diff
@@ -94,3 +94,17 @@ nav ul li a {
 .cart_total {
   float: right;
 }
+
+tr input[type=text] {
+  margin-right: -20px;
+  width: 50px;
+  text-align: center;
+}
+
+.btn-small {
+  margin-right: 50px;
```

Comment:
Properties should be ordered font-size, height, line-height, margin-right, padding